### ISSUE TITLE
SLAM with namespaced Robots

### ISSUE THREAD

**Issue by ipa-rar** [turtlebot/turtlebot4#159]
**Describe the the feature you would like**
Documentation on using namespace robots with the turtlebot4_navigation. This is not found in the ``multiple_robots.md`` and its very critical for multirobot users.

**Other notes**
Add anything else you thing is important.

**Comment by roni-kreinin** [comment]
We will add that. You can use the `namespace` launch argument to launch navigation for specific robots.

`ros2 launch turtlebot4_navigation slam.launch.py namespace:=/robot1`

**Comment by ipa-rar** [comment]
@roni-kreinin What other changes has to be made to the [parameters](https://github.com/turtlebot/turtlebot4/tree/humble/turtlebot4_navigation/config) inorder to get it running. In my case I have frame_prefix to the links.

**Comment by roni-kreinin** [comment]
Are you trying to generate a single map using multiple robots? Or a separate map for each robot?

**Comment by ipa-rar** [comment]
> A single map with a single robot

A single map that can be shared by multiple robots once mapping is complete. Each robot will be using the same map for navigation within a closed arena. 

**Some  info regarding my setup**
After adding namespace during the robot setup all the topics had namespace including ``/tf``and ``/tf_static``. However the ``namespace1/tf`` and ``namespace2/tf`` had the same links and did not have a frame_prefix which corresponds to the namespace of the robot. So I created a relay node that listens to all ``namespaceX/tf`` and ``namespaceX/tf_static`` topics and add a `frame_prefix`  which is `namespaceX` and then relay it back to `/tf` and `/tf_static`. 

![Screenshot from 2023-05-03 18-08-38](https://user-images.githubusercontent.com/49311464/235975751-4aaef32a-39c7-455e-a788-6506ae4bbd33.png)


The image contains just one robot called `agent_1` with `frame_prefix`= `agent_1`

**Comment by roni-kreinin** [comment]
We don't yet have any examples of how to run this, but that does seem to be the correct approach. If you are able to get this running then feel free to make a PR to https://github.com/turtlebot/turtlebot4_tutorials and we can add a tutorial.

**Comment by hsd-dev** [comment]
Wouldn't things be easier if the turtlebot Xacros include `prefix` parameter for links and joints names? That way each turtlebot can be launched with its specific prefix.

**Comment by andersonjwan** [comment]
I agree with this. Currently, the issue lies in that the URDF does not inherently support prefixes in the links created. Therefore, after running the xacro command, the resulting frame identifiers are the same regardless of the namespace.
